thirdeye/thirdeye-pinot/src/main/java/com/linkedin/thirdeye/anomaly/classification/ClassificationJobRunner.java

```diff
@@ -35,6 +35,7 @@ public Long createJob() {
       JobDTO jobSpec = new JobDTO();
       String jobName = createJobName(jobContext);
       jobSpec.setJobName(jobName);
+      jobSpec.setConfigId(jobContext.getConfigDTO().getId());
```

Comment:
rename the configDTO in classificationJobContext to make it clear what config this is talking about